doc/classes/SpinBox.xml

```diff
@@ -56,15 +56,22 @@
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable" default="true" keywords="readonly, disabled, enabled">
 			If [code]true[/code], the [SpinBox] will be editable. Otherwise, it will be read only.
 		</member>
-		<member name="prefix" type="String" setter="set_prefix" getter="get_prefix" default="&quot;&quot;">
+		<member name="format" type="String" setter="set_format" getter="get_format" default="&quot;&quot;">
+			Specifies formatting of the displayed numeric value, using the same rules as operator [code]%[/code] in [String]. Only applies when the value is not being currently edited. If empty, the value will use default display. Using [code]%s[/code] placeholder will display the value with default formatting. If the format is invalid, the [SpinBox] will show "ERROR", with the actual error message available in the configuration warning.
+			Examples:
+			[code]%0.2f%%[/code] - displays as percentage, with 2 decimal points, e.g. [code]21.37%[/code].
+			[code]%d ms[/code] - displays as integer with milliseconds unit, e.g. [code]420 ms[/code].
+			[code](%s)[/code] - displays as a number in brackets, e.g. [code](6.9)[/code].
```

Comment:
Formatting the format strings. Also shuffling the order from simplest to most intricate example(?).

```suggestion
			The following format strings can be used as examples:
			- [code]"(%s)"[/code] displays as a number in brackets (e.g. [code](6.9)[/code]).
			- [code]"%d ms"[/code] displays as an integer with milliseconds unit (e.g. [code]420 ms[/code]).
			- [code]"%0.2f%%"[/code] displays as a percentage, with 2 decimal points (e.g. [code]21.37%[/code]).
			
```
Or similar

I'm not sure whether to keep the quotation marks or not. It's how strings _should_ be represented in the docs, but there's potential ambiguity because users would be inclined to copy and paste them, as well.